_quarto.yml

```diff
@@ -13,19 +13,19 @@ website:
           - href: people.qmd
           - href: join_us.qmd
           - href: contact_us.qmd
-      - href: news.qmd
-        text: "News"
+      - text: "News"
+        menu:
+          - href: news.qmd
```

Comment:
since currently there is just one entry, I would just do like before , maybe even drop the events for now? thinking about it, I guess events would mostly be presentations or webinars/videos, and we have other categories for those anyway already?

Reply:
agreed and changed it back.